### PR TITLE
(init) method to create 10mil fake data and import to mongoDB

### PR DESCRIPTION
**Pull request by lee-philip-n**
https://trello.com/c/P0Pw8lnT
https://trello.com/c/JfmbWM47

### CODE REVIEW

File: client/src/SummaryView.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ class SummaryView extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      id: props.id,
+      id: this.props.id,
       name: '',
```
Comment (matgbx) [inline client/src/SummaryView.jsx:12]: good catch here

File: client/src/ajax.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const BASE_URL = process.env.BASE_URL ? process.env.BASE_URL : '';
 
 const get = (id, cb) => {
   $.ajax({
-    url: `${BASE_URL}/restaurants/${id}`,
+    url: `/restaurants/${id}`,
     method: 'GET',
```
Comment (matgbx) [inline client/src/ajax.js:8]: why did you remove the base URL?

File: createDataFile.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        price: faker.Helpers.randomNumber(100),
+        style: faker.Lorem.words(),
+        phone: faker.PhoneNumber.phoneNumber(),
+      },
+      banner: [faker.Image.nightlife(), faker.Image.nightlife(), faker.Image.nightlife(), faker.Image.nightlife()],
+      photo: [faker.Image.food(), faker.Image.food(), faker.Image.food()],
+    };
+    container.push(data);
+    i++;
+    if (i !== 0 && i % 500000 === 0) {
```
Comment (matgbx) [inline createDataFile.js:23]: Good way of splitting the write task in to chunks

**Review by matgbx — APPROVED** [review]
All looks good. One question in there about the removal of the base url in the client ajax call, but that's all good.

File: client/src/ajax.js (hunk 2 above)
Comment (lee-philip-n) [reply client/src/ajax.js:8]: the ajax url should be a relative url, so ${BASE_URL} is not required.
